bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -934,6 +938,17 @@ lnd/clm2/surfdata_map/surfdata_ne120np4_78pfts_CMIP6_simyr1850_c170824.nc</fsurd
 <flanduse_timeseries hgrid="ne120np4"      rcp="8.5" sim_year_range="1850-2100" 
  use_crop=".true."  >lnd/clm2/surfdata_map/landuse.timeseries_ne120np4_hist_78pfts_CMIP6_simyr1850-2015_c170824.nc</flanduse_timeseries>
 
+<flanduse_timeseries hgrid="C24"      rcp="8.5" sim_year_range="1850-2100"
```

Comment:
Same remove these...

Reply:
So no 1850-2100 SSP files then?

Reply:
Whoops, sorry I thought these were another set of fsurdat files.

I'd like to include one SSP5-8.5 landuse timeseries for the C96 case and also point to the same file for the historical case.

For the other files, we want to limit our support for landuse.timeseries files since they are large. So the low resolution files are probably fine. The high resolution cases might be large enough that we don't really want to make them, unless you know they are absolutely needed. And other than the C96 landuse.timeseries, we won't automatically create them when we update surface datasets, but ask you to do so.

Reply:
@ekluzek I got rid of everything except the 1 degree and a low resolution (C24 - 4 degree) dataset.

Reply:
Perfect.